code/obj/storage/secure_closets.dm

```diff
@@ -89,6 +89,7 @@
 	/obj/item/clothing/suit/armor/vest,
 	/obj/item/stamp/hop,
 	/obj/item/device/radio/headset/command/hop,
+	/obj/machinery/computer/card/portable,
```

Comment:
Meh about this in lockers, at least in part because putting it back in the locker will probably not work as it's not an `/obj/item`. They are also anchored and block movement.

Reply:
its now a manufacturer item, the anchored is the norm for other items similar to it, it no longer blocks movements.